tests/simple-from-il-binary/simple.cl

```diff
@@ -1,5 +1,10 @@
+struct ParamStruct {
+    char a;
+};
+
 kernel void test_simple(global uint* out)
 {
+    struct ParamStruct a;
```

Comment:
Fine to leverage the simple test for this for now as a shortcut but if we find more of these SPIR-V related issues, we'll need to extend the unit test framework instead. I'll capture this in an issue once when I merge this PR.